### PR TITLE
Remove Pedometer Resume and Pause methods

### LINKED ISSUE #968: Pedometer Pause = Pedometer Stop
Pedometer.Stop is just a call to Pedometer.Pause. Having two separate buttons for this implies different functionality. I propose that we simply remove the "Stop" function and all current uses of Stop can just become calls to Pause. 
Similarly, "Resume"....is a call to Start. Resume should probably be removed as well. 

```
public void Start() {
    if (pedometerPaused) {
      pedometerPaused = false;
      sensorManager.registerListener(this,
          sensorManager.getSensorList(Sensor.TYPE_ACCELEROMETER).get(0),
          SensorManager.SENSOR_DELAY_FASTEST);
      startTime = System.currentTimeMillis();
    }
  }

  /**
   * Stops the pedometer.
   */
  @SimpleFunction(description = "Stop counting steps")
  public void Stop() {
    Pause();
  }

  /**
   * Resumes the counting of steps.
   */
  @SimpleFunction(description = "Resumes counting, synonym of Start.")
  public void Resume() {
    Start();
  }

  /**
   * Pauses the counting of steps.
   */
  @SimpleFunction(description = "Pause counting of steps and distance.")
  public void Pause() {
    if (!pedometerPaused) {
      pedometerPaused = true;
      sensorManager.unregisterListener(this);
      Log.d(TAG, "Unregistered listener on pause");
      prevStopClockTime += (System.currentTimeMillis() - startTime);
    }
  }
```

### PR DESCRIPTION
**Pull request by bartmathijssen**
Fixes #968

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

File: appinventor/components/src/com/google/appinventor/components/runtime/Pedometer.java [hunk 1]
```diff
@@ -132,27 +137,6 @@ public void Reset() {
     startTime = System.currentTimeMillis();
   }
 
-  /**
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/Pedometer.java]: Any particular reason why these definitions were moved further down in the file? You could keep them here while replacing the body.

**Comment by ewpatton** [comment]
@AppInventorWorkerBee ok to test

File: appinventor/components/src/com/google/appinventor/components/runtime/Pedometer.java (hunk 1 above)
Comment (bartmathijssen) [reply appinventor/components/src/com/google/appinventor/components/runtime/Pedometer.java]: While looking at `Pedometer.java` I noticed there was a section for deprecated properties and events. I thought I might as well at these new deprecated methods to this section. I have reversed the moving of the methods in the latest commit.

**Comment by SusanRatiLane** [comment]
@bartmathijssen @ewpatton I am still seeing the deprecated methods in the blocks flyout for a new Pedometer component. Could someone else confirm?

**Comment by bartmathijssen** [comment]
@SusanRatiLane I also noticed this while developing but forgot to mention it when creating the pull request. To my knowledge, I properly completed all the steps to deprecate a function. Perhaps this is an issue regarding the Deprecated annotation not working properly.

**Comment by SusanRatiLane** [comment]
@bartmathijssen It certainly looks correct in the code. I'll do more testing of the annotation.

**Comment by ewpatton** [comment]
@SusanRatiLane @bartmathijssen This one is on me. Once upon a time I did what I thought was a reasonable thing by making it so that the deprecated flag was a Boolean rather than a string, but that broke some logic which tested for a string `"true"` rather than the Boolean `true`. This is corrected in #1923, which when applied to this PR gives the expected behavior.

**Pull request merged**